glide.yaml

```diff
@@ -1,23 +0,0 @@
-package: github.com/Shopify/ghostferry
-import:
-- package: github.com/Masterminds/squirrel
```

Comment:
looks like we're missing squirrel in Gopkg.toml.. not sure how tests are passing, maybe because we have an indirect dependency?